client/my-sites/upgrades/components/form/country-select.jsx

```diff
@@ -70,11 +70,18 @@ module.exports = React.createClass( {
 						onChange={ this.props.onChange }
 						onClick={ this.recordCountrySelectClick }
 						isError={ this.props.isError } >
-						{ options.map( function( option ) {
-							return <option key={ option.key } value={ option.value || '' } disabled={ option.disabled }>{ option.label }</option>;
+						{ options.map( option => {
```

Comment:
You ca use parentheses to make use of the implicit return, so instead of this:
```js
options.map( option => {
  return <option...
```
do this:
```js
options.map( option => (
  <option...
) )
```